packages/shared/test-utils/src/fastify-test-app.ts

```
@@ -0,0 +1,36 @@
+import 'reflect-metadata'
+import type { FastifyInstance } from 'fastify'
+import Fastify from 'fastify'
+import { setupContainer } from '../../../apps/api/src/shared/di/container'
+
+interface TestAppOptions {
+  logger?: boolean
+  modules?: Array<{
+    module: FastifyPluginAsync
```

ESLint

[error] 'FastifyPluginAsync' is not defined.

(no-undef)